src/log-store/src/raft_engine/backend.rs

```diff
@@ -41,9 +42,9 @@ pub struct RaftEngineBackend {
 
 #[async_trait::async_trait]
 impl TxnService for RaftEngineBackend {
-    type Error = error::Error;
+    type Error = meta_error::Error;
```

Comment:
Does the error type has to be changed to `meta_error::Error`? It introduces too much snafu context inside. A little ugly.

Reply:
It's better to use `meta_error::Error`, To avoid having too many `KvBackendAdapter` s.